script.js

```diff
@@ -31,4 +130,64 @@ function closewindow() {
   closebutton.classList.add('hidden');
 }
 
-navigationLinks.addEventListener('click', closewindow);
\ No newline at end of file
+navigationLinks.addEventListener('click', closewindow);
+
+for (let i = 0; i < projects.length; i += 1) {
+  const projectCard = document.createElement('article');
+  projectCard.classList.add('projectCard');
+  const cardIMG = document.createElement('div');
+  cardIMG.classList.add('cardIMG');
+  cardIMG.innerHTML = `<img src="${projects[i].featureImage}" alt="Project 2">`;
+  const cardContent = document.createElement('div');
+  cardContent.classList.add('cardContent');
+  cardContent.innerHTML = `
+  <h3 class="cardTitle textWhite">${projects[i].name}</h3>
+  <p class="cardDescription textWhite">${projects[i].description}
+  </p>
+  <ul class="boxes">
+  ${projects[i].tecnologies.map((element) => `<li class="textWhite">${element}</li>`).join('')}
+  </ul>
+  <button id="projectButton${i}" class="projectButton open">See project</button>
+  `;
+}
+
+open.addEventListener('click', () => {
+  modalContainer.classList.add('show');
+  modaltitle.textContent = 'Multi-Post Stories';
+  modalDescription.textContent = `A daily selection of privately personalized reads; no accounts or sign-ups required.
+    has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.`;
+  modalTecnologies.innerHTML = `<li>CSS</li>
+    <li>HTML</li>
+    <li>Bootstrap</li>
+    <li>Ruby</li>`;
+  liveDemo.href = 'https://yulianav09.github.io/Portfolio/';
+  source.href = 'https://github.com/Yulianav09/Portfolio';
+  snapshoot.src = './src/Img Placeholder works.png';
+  body.classList.add('blockScroll');
+});
+
+closex.addEventListener('click', () => {
+  modalContainer.classList.remove('show');
+  body.classList.remove('blockScroll');
+});
+
+// validation email
+
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const formControl = document.querySelector('.formControl');
+
+function checkInputs() {
+  const emailValue = email.value.trim();
+
+  if (emailValue !== emailValue.toLowerCase()) {
+    formControl.classList.add('error');
+  } else {
+    formControl.classList.remove('error');
+  }
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  checkInputs();
+});
```

Comment:
- [ ] Great job on implementing the project requirement! 👏 However, there is a minor issue that we need to address before approving your project. 
  - Your form validation is working fine but what seems not to be working as expected is the summit `button`, When a user enters an invalid `email` they get an error `message` which is nice and fine but they enter the correct `email` without uppercase letters in it and clicks the submit `button` the form doesn't get submitted.
  - This is happening because you're not calling the `submit()` function on the form. Kindly consider calling the `submit()` function on your form in your event listener to have users submit their forms successfully.